ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-base.md

```diff
@@ -193,4 +209,8 @@ router bgp 65101
       network 2001:db8:100::/40
       network 2001:db8:200::/40 route-map RM-BAR-MATCH
       redistribute static route-map RM-IPV6-STATIC-TO-BGP
+  session tracker ST1
```

Comment:
These commands look incorrectly indented to me, only 2 instead of 3 spaces.

Reply:
Good catch - addressed in 76c6075cf